demo/midgard-sdk/src/common.ts

```diff
@@ -243,6 +251,38 @@ export const AddressSchema = Data.Object({
 export type AddressData = Data.Static<typeof AddressSchema>;
 export const AddressData = AddressSchema as unknown as AddressData;
 
+export const fromAddress = (
+  address: Address,
+): Effect.Effect<AddressData, DataCoercionError> =>
+  Effect.gen(function* () {
+    const { paymentCredential, stakeCredential } = getAddressDetails(address);
```

Comment:
This is a function that can throw, so it should be wrapped under `Effect.try()`.